site/operational-guide/concepts.md

```diff
@@ -101,11 +102,11 @@ scenarios:
 
 ### Consensus
 
-Alternatively, when a new network is formed for the first time peers
+Alternatively, when a new network is formed for the first time, peers
 can be configured to co-ordinate amongst themselves to automatically
 divide up the IP allocation range. This process is known as consensus
-and requires each peer to be told the total number of expected peers
-(the 'initial peer count') in order to prevent formation of disjoint
+and it requires each peer to be told the total number of expected peers
+(the 'initial peer count') in order to prevent the formation of disjointed
```

Comment:
we are talking about the mathematical disjointness here, so I think it should be 'disjoint'. See http://english.stackexchange.com/questions/122069/disjoint-toolsets-vs-disjointed-toolsets